p2p/src/authenticated/discovery/actors/peer/actor.rs

```diff
@@ -71,14 +71,18 @@ impl<E: Spawner + Clock + ReasonablyRealtime + Rng + CryptoRng + Metrics, C: Pub
     }
 
     /// Unpack `msg` and verify the underlying `channel` is registered.
-    fn validate_msg<V>(msg: Option<Data>, rate_limits: &HashMap<u32, V>) -> Result<Data, Error> {
+    fn validate_outbound_msg<V>(
```

Comment:
I did a double take and thought this was checking incoming messages were on a valid channel, so figured it would be clearer to rename.